### PR TITLE
Use user-defined fixed rate and eliminate random sleep duration

### PR DESCRIPTION
**Pull request by padmashree9110**
1. Used rate instead of duration sleep.
2. Changed loop count accordingly.
- [X] Addresses #13

### CODE REVIEW

File: ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp [hunk 1]
```diff
@@ -35,11 +35,12 @@ void AeroDrone::fetchTelemetryThread()
   // Battery state
   auto battery_state_sub = nh_.subscribe<sensor_msgs::BatteryState>(
       "mavros/battery", 1, boost::bind(&AeroDrone::setBatteryStateCB, this, _1));
+  ros::Rate rate(10);
```
Comment (shakthi-prashanth-m) [inline ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp:38]: I think you can remove `rate_` from [AeroDrone class](https://github.com/intel-aero/sample-apps/blob/master/ros/src/aero_telemetry_simple/include/aero_drone_telemetry.h#L45).

**Review by shakthi-prashanth-m — COMMENTED** [review]
Seems ok; with a change.

File: ros/src/aero_telemetry_simple/include/aero_drone_telemetry.h.autosave [hunk 2]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (shakthi-prashanth-m) [inline ros/src/aero_telemetry_simple/include/aero_drone_telemetry.h.autosave]: Remove this file.

**Review by shakthi-prashanth-m — CHANGES_REQUESTED** [review]

File: ros/src/aero_offboard_velocity/src/aero_drone_offboard.cpp [hunk 3]
```diff
@@ -163,6 +164,7 @@ void AeroDrone::setOffboardVelocityNED(float vx, float vy, float vz, float yaw,
   for (; count > 0; count--)
   {
     set_vel_pub_.publish(pos);
-    ros::Duration(0.01).sleep();
+    rate_.sleep();
+    // ros::Duration(0.01).sleep();
```
Comment (anitha-suresh-zz) [inline ros/src/aero_offboard_velocity/src/aero_drone_offboard.cpp]: Please remove commented line

File: ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp [hunk 4]
```diff
@@ -35,11 +35,12 @@ void AeroDrone::fetchTelemetryThread()
...
 
   while (ros::ok())
   {
     ros::spinOnce();
-    rate_.sleep();
+    rate.sleep();
```
Comment (anitha-suresh-zz) [inline ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp:43]: Are you trying to use rate_

**Review by anitha-suresh-zz — CHANGES_REQUESTED** [review]

File: ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp (hunk 4 above)
Comment (padmashree9110) [reply ros/src/aero_telemetry_simple/src/aero_drone_telemetry.cpp:43]: There is no such variable `rate_` in telemetry example. What we have used is local variable [rate](https://github.com/intel-aero/sample-apps/pull/14/files/4f96d3b239a8e288b3119f646b890a2609820220#diff-0ae7a01ee16f76ff11392b4ca86a6c98R38).

**Review by shakthi-prashanth-m — APPROVED** [review]

**Comment by shakthi-prashanth-m** [comment]
@anitha-suresh I am done with the review. You can review and merge.

**Review by anitha-suresh-zz — APPROVED** [review]

**Pull request merged**